aerogear/mobile-client-service
# Aerogear 7675

### Changes:

* Add handler to watch for secret changes and to update the CRD based on these changes.
* Updated tests

### Verification Steps:
* Create a mobile client
* Create a secret with the appropriate labels, an example can be seen here: https://gist.github.com/StephenCoady/ca16dfa3b3be4a399fd57a272662ccd4
* Browse to the mobile app page and the mobile client config should contain the new service you linked to previously
* Edit the secret and change the id field
* Return to the mobile app page and the service should be updated to reflect this change
* Delete this secret
* Return to the mobile app page and the service should be removed from the client config

### JIRA:

https://issues.jboss.org/browse/AEROGEAR-7675
